bundles/org.openhab.binding.emby/src/main/java/org/openhab/binding/emby/internal/handler/EmbyDeviceHandler.java

```diff
@@ -0,0 +1,333 @@
+                        logger.debug("Sending the following payload: {} for device: {}", command.toString(),
+                                currentSessionID);
+                        handler.sendCommand(CONTROL_SESSION + currentSessionID + CONTROL_SENDPLAY, command.toString());
+                        break;
+
+                    case CHANNEL_GENERALCOMMAND:
+                        commandName = channel.getConfiguration().get(CHANNEL_GENERALCOMMAND_NAME).toString();
+                        logger.debug("Sending the following command {} for device: {}", commandName, currentSessionID);
+                        if (OnOffType.ON.equals(command)) {
+                            handler.sendCommand(
+                                    CONTROL_SESSION + currentSessionID + CONTROL_GENERALCOMMAND + commandName);
+                        }
+                        break;
+                    case CHANNEL_GENERALCOMMANDWITHARGS:
+                        commandName = channel.getConfiguration().get(CHANNEL_GENERALCOMMAND_NAME).toString();
+                        logger.debug("Sending the following command {} for device: {}", commandName, currentSessionID);
+                        handler.sendCommand(CONTROL_SESSION + currentSessionID + CONTROL_GENERALCOMMAND + commandName,
+                                "Arguments:" + command + "}");
+                        break;
+                }
+            }
+
+        } else {
+            updateStatus(OFFLINE, CONFIGURATION_ERROR,
+                    "Unable to handle command, You must choose a Emby Server for this Device.");
+        }
+    }
+
+    private EmbyDeviceConfiguration validateConfiguration() {
+        EmbyDeviceConfiguration embyDeviceConfig = new EmbyDeviceConfiguration(
+                this.thing.getConfiguration().get(DEVICE_ID).toString());
+        Configuration testConfig = this.thing.getChannel(CHANNEL_IMAGEURL).getConfiguration();
+        if (!(testConfig.get(CHANNEL_IMAGEURL_MAXWIDTH) == null)) {
+            embyDeviceConfig.imageMaxWidth = testConfig.get(CHANNEL_IMAGEURL_MAXWIDTH).toString();
+        }
+        if (!(testConfig.get(CHANNEL_IMAGEURL_MAXHEIGHT) == null)) {
+            embyDeviceConfig.imageMaxHeight = testConfig.get(CHANNEL_IMAGEURL_MAXHEIGHT).toString();
+        }
+        String testPercentPlayed = "true";
+        if (!(testConfig.get(CHANNEL_IMAGEURL_PERCENTPLAYED) == null)) {
+            testPercentPlayed = testConfig.get(CHANNEL_IMAGEURL_PERCENTPLAYED).toString();
+        }
+
+        embyDeviceConfig.imageImageType = testConfig.get(CHANNEL_IMAGEURL_TYPE).toString();
+
+        if ((testPercentPlayed.equals("true"))) {
+            embyDeviceConfig.imagePercentPlayed = true;
+        } else {
+            embyDeviceConfig.imagePercentPlayed = false;
+        }
+
+        return embyDeviceConfig;
+
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing emby device: {}", this.getThing().getLabel());
```

Comment:
I am confused by what this means?  I would think having this would be helpful in log files for troubleshooting and it is set on the debug level?  But is this not the right way to do this?